packages/next-urql/src/with-urql-client.tsx

```diff
@@ -29,9 +29,10 @@ export function withUrqlClient(
   options?: WithUrqlClientOptions
 ) {
   if (!options) options = {};
+
   return (AppOrPage: NextPage<any> | typeof NextApp) => {
-    const shouldBindGetInitialprops = Boolean(
-      AppOrPage.getInitialProps || options!.ssr
+    const shouldEnableSuspense = Boolean(
+      AppOrPage.getInitialProps || (options!.ssr && !options!.neverSuspend)
```

Comment:
This might cause a situation where `AppOrPage.getInitialProps` truthy and will override the `neverSuspend`?

Reply:
Good spot, prettier goofed up brackets